packages/back-end/src/routers/saved-group/saved-group.controller.ts

```diff
@@ -480,6 +495,8 @@ export const putSavedGroup = async (
     fieldsToUpdate.description = description;
   }
 
+  fieldsToUpdate.projects = projects;
```

Comment:
Do we want to check here that the user has permission to create within all the projects in `fieldsToUpdate`?